jobs/official_release.groovy

```diff
@@ -24,6 +24,8 @@ p.pipeline().with {
     stringParam('SPLENV_REF', scipipe.template.splenv_ref, 'conda env ref or eups tag')
     stringParam('RUBINENV_VER', scipipe.template.splenv_ref, 'rubin-env version')
     booleanParam('O_LATEST', false,
-      'update the eups "O_LATEST" tag -- should only be done for a final (non-rc) release')
+      'update the eups "O_LATEST" tag -- should only be done for a final (non-rc) release'),
+    stringParam('EXCLUDE_FROM_TARBALLS', null,
+      'products to exclude from tarball builds. Eg. "lsst_sitcom"')
```

Comment:
```suggestion
      'products to exclude from tarball builds, e.g. "lsst_sitcom"')
```

Reply:
Updated all other occurrences to proper English